minutes/2019 Weekly & Monthly Sync.md

```diff
@@ -0,0 +1,2997 @@
+    *   D&I Badging
+    *   Community Reports
+        *   Zephyr 
+        *   JenkinsX 
+    *   Website 
+    *   Social Media // Communication 
+        *   Content group created 
+        *   Moving forward 
+    *   Mentorship Program
+        *   [https://funding.communitybridge.org/projects/chaoss](https://funding.communitybridge.org/projects/chaoss)
+        *   Outreachy 
+            *   Pre-application period for projects is Late January, early February 
+            *   AI: Matt Do This 
+*   Outreach 
+    *   Linux Foundation Events
+        *   Reconnecting 
+    *   Apache Software Foundation Survey
+        *   Connection with CHAOSS -- Community survey is delayed but this may begin very shortly 
+*   CHAOSScon 
+    *   January 31, 2020
+    *   Process of selecting the talks 
+    *   Meeting later this week to make decisions on the talks 
+    *   Notifications likely in the second week of December 
+    *   Registration 
+    *   Ping Deb for a talk Title
+    *   Perhaps add another Keynote 
+
+
+## November 26, 2019
+
+Attendees: Matt G., Ray, Dawn, Matt S., Daniel, Saleh, Kevin L. Armstrong, Ildiko, Vinod, 
+
+Video: [https://youtu.be/pgyBdGvOSvY](https://youtu.be/pgyBdGvOSvY) 
+
+Notes:
+
+Community Reports
+
+
+
+*   Work being done by GL folks for JenkinsX
+    *   GL screen that came from the work on the JenkinsX report. In essence, bringing last weeks drawings to life during the GL weekly meeting
+        *   [https://chaoss.biterg.io/app/kibana#/dashboard/5b0b0c90-106a-11ea-9c54-4fefc638758f](https://chaoss.biterg.io/app/kibana#/dashboard/5b0b0c90-106a-11ea-9c54-4fefc638758f)
+*   Connecting with UNO folks to do this work 
+*   Saleh can help out too. 
+
+Community Bridge 
+
+
+
+*   They cannot do business to business transactions 
+*   Folks will have to pay out of pocket and then get reimbursed
+*   Mixing some dollars at times 
+*   Need to make donations in whole dollar amounts 
+
+CHAOSScon: 
+
+
+
+*   Submissions: 18 submissions (about 10 relevant submissions) 
+*   We would like to improve diversity
+*   Possibly extending deadline
+*   Max number of talks? 
+*   Perhaps more than one keynote 
+*   Planning Doc: 
+    *   [https://docs.google.com/document/d/1HvKPfJwoZm4nW2bBVbDfN8HyFXyXsJlN9g16UhR6l-A/edit#heading=h.k0knjt1mmdb1](https://docs.google.com/document/d/1HvKPfJwoZm4nW2bBVbDfN8HyFXyXsJlN9g16UhR6l-A/edit#heading=h.k0knjt1mmdb1) 
+
+
+
+<p id="gdcalert1" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image1.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert2">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
```

Comment:
It looks like there should be a way to get the embedded images. Could you see if you can add them to a sub-directory here in the repo and embed them in the Markdown?

Reply:
I fixed this, but I had trouble merging because it is open in a PR here.

@mpgjon I could provide you with the raw md file that includes the image links if that works

Reply:
Got it @Nebrethar  - send me the raw MD and I can edit and resubmit?  Thanks!

Reply:
Sent over Slack :)